evals/swe_bench/build_images.py

```diff
@@ -7,16 +7,17 @@
 from docker.client import DockerClient  # type: ignore
 from swe_bench import SAMPLE_TO_IMAGE_PATH  # type: ignore
 from swebench.harness.docker_build import build_env_images  # type: ignore
-from swebench.harness.test_spec import get_test_specs_from_dataset  # type: ignore
+from swebench.harness.test_spec import get_test_specs_from_dataset, make_test_spec  # type: ignore
 from swebench.harness.utils import load_swebench_dataset  # type: ignore
+from datasets import load_dataset
+from inspect_ai.dataset import Sample
 
 
```

Comment:
This now goes from list[Sample] -> built images, and a dictionary for those images.